pom.xml

```diff
@@ -463,14 +485,19 @@
             </plugin>
             <plugin>
                 <artifactId>maven-surefire-plugin</artifactId>
-                <version>2.22.2</version>
                 <configuration>
+                    <excludes>
+                        <exclude>${pattern.class.test.int}</exclude>
+                    </excludes>
                     <includes>
-                        <include>**/*Test.java</include>
-                        <include>**/*Tests.java</include>
-                        <include>**/*Test_*.java</include>
-                        <include>**/*Tests_*.java</include>
+                        <include>${pattern.class.test}</include>
```

Comment:
Note that the new pattern doesn't include some of the tests with the naming convention  `SomeTest_WithSomeSpecificSituation`.

Reply:
Gotcha. I'll add it back in :+1: